### ISSUE TITLE
BMK visit 06 March: Bugs & todo's

### ISSUE THREAD

**Issue by patrick-griffiths** [eurodatacube/eodash#2052]
@lubojr and @santilland 

I created this issue to track bugs, glitches and other to-do's until next Monday's visit to the BMK

Bug/Glitches: 
- [x] Renewable tools: Distance to Settlements filter does not work currently
- [x] Renewable tools: Exclude protected areas does not work currently

**Comment by patrick-griffiths** [comment]
Wishlist with smaller items:
- [ ] Renewable tools: could we have a "reset analysis" button that resets all constraint sliders to the default?

**Comment by patrick-griffiths** [comment]
> @lubojr and @santilland
> 
> I created this issue to track bugs, glitches and other to-do's until next Monday's visit to the BMK
> 
> Bug/Glitches:
> 
> * [x]  Renewable tools: Distance to Settlements filter does not work currently
> * [x]  Renewable tools: Exclude protected areas does not work currently

dear @lubojr thanks for looking into this.. 
I checked the functionality again, and the distance to settlement slider is still without effect, also after a hard reload..
Maybe we could connect briefly this morning to take a look together.. Feel free to send me an invite for anytime before 13.00 (of course we can also connect tomorrow if needed)

**Comment by lubojr** [comment]
Dear @patrick-griffiths we have not deployed these changes to production. Please check our new staging page https://gtif-staging.eox.at

**Comment by lubojr** [comment]
@patrick-griffiths Yes, I did not understand the implication/usecase of this variable. I thought that we aim to show areas near the settlements, while instead it is to show the opposite. Sorry.

I have changed the behavior now that if this filter is enabled, we only show areas with distance value above the selected value.

I have updated the max range to 5000m, but to be honest, the largest value of the raster is 5040m and this is a single pixel in the middle of alps. If this is a data problem or not, I do not know.

**Comment by patrick-griffiths** [comment]
hi @lubojr  @santilland 

it seems the updated from https://gtif-staging.eox.at/
have not been deployed to the https://gtif.esa.int/ environment

When will this happen?

**Comment by patrick-griffiths** [comment]
hi @lubojr @santilland 

any updates on the first integration of the hydro reservoir monitoring capability?

**Comment by santilland** [comment]
@DanGasb as for the item:
2. Innsbruck hot-spot, surface NO2 concentrations --> add the units ; ppbv. Why there are data only from September 2022? We should have data from January 2020.
--> the units have been added, as for the data, the whole time series is loaded into geodb, we are currently showing a subset, as showing all the data at once is too much to handle by the client, and would make the chart quite unreadable.
We need to design and implement a concept for showing large time series in plots, see created issue https://github.com/eurodatacube/eodash/issues/2013. This will take some time.
Related to "replace the blue square with a pointer", similar situation, we do not have the concept of showing a "needle" as area of interest, we consider this as a "nice to have" or enhancement which can't take priority right now (issue created here https://github.com/eurodatacube/eodash/issues/2058).
4. Working on increasing logo

**Comment by santilland** [comment]
> hi @lubojr @santilland
> 
> it seems the updated from https://gtif-staging.eox.at/ have not been deployed to the https://gtif.esa.int/ environment
> 
> When will this happen?

I just merged some updates to staging (minutes ago) - carbon accounting congids - today we will increase our deployment to the beta deployment, but in the hours range (minimum)

**Comment by santilland** [comment]
> hi @lubojr @santilland
> 
> any updates on the first integration of the hydro reservoir monitoring capability?

Working on it, expecting to have it ready today at some point, will keep you posted

**Comment by lubojr** [comment]
@patrick-griffiths First implementation of REP4 indicator over stausee is on https://gtif-testing.eox.at/gtif_wse_swe/

I still plan to add:
- [ ] the second dataset (WSE) as another indicator in the list on the left (due to different times of measurement they can not be properly shown as one indicator) 
- [ ] tooltip over value in chart would show the absolute value `area` in km2

**Comment by santilland** [comment]
@DanGasb DLR logo updated with one of same format as the others, all 3 logos now share the same width. Crossed of point 4. in previous comment.
![image](https://user-images.githubusercontent.com/4036819/222767230-24d498a9-ef4b-408c-84b9-876c9b575a0b.png)

**Comment by santilland** [comment]
@patrick-griffiths slider for distance to energy grid has been changed to single value selection for both wind and solar energy

**Comment by santilland** [comment]
> Wishlist with smaller items:
> 
> * [ ]  Renewable tools: could we have a "reset analysis" button that resets all constraint sliders to the default?

Other priorities were taken for data integration and configuration, created new issue to keep track of enhancement https://github.com/eurodatacube/eodash/issues/2063

**Comment by patrick-griffiths** [comment]
> * nd dataset (WSE) as another indicator in the list on the left (due to different dates of measurement they can not be properly shown as one indicator)



> @patrick-griffiths slider for distance to energy grid has been changed to single value selection for both wind and solar energy



> @patrick-griffiths First implementation of REP4 indicator over stausee is on https://gtif-testing.eox.at/gtif_wse_swe/
> 
> I still plan to add these two points tomorrow morning:
> 
> * [ ]  the second dataset (WSE) as another indicator in the list on the left (due to different dates of measurement they can not be properly shown as one indicator)
> * [ ]  tooltip over value in chart would show the absolute value `area` in km2

ok nice I see it in the testing env @lubojr , however there was two lake level measurement time series, sheet SWE and WSE in the spreadsheet. I do not see both of them. Not a big issue. Great to see something at least.. 

Anyway could you please:
- [ ] Reorder the reneable tools, so that we have: wind, solar, hydro, microhydro, nowcasting

**Comment by patrick-griffiths** [comment]
@lubojr and @santilland 

could you push the hydro reservoir into the demo environment please? 
And please update the order of the energy tools as indicated above (wind, solar, hydro, microhydro, nowcasting)

**Comment by lubojr** [comment]
@patrick-griffiths I have merged the current state of SWE into the deployed version. One point has been addressed (the hover operation on point in chart shows a tooltip with the rest of the data which were present in the original sheet).

I would still be able to integrate the WSE as well, but due to time mismatch of WSE and SWE as a completely separate dataset - it would be selectable in parallel to `Hydro Power` now. Would you prefer that or rather keep it as is for tomorrow?
 
![image](https://user-images.githubusercontent.com/19269667/222962493-2ca858f6-a681-4f3b-90ee-4fec9ca43a6c.png)

**Comment by lubojr** [comment]
Closing the issue as all points have been either moved to a separate issue or fixed
